src/retasc/constants.py

```diff
@@ -0,0 +1,9 @@
+"""
+Constants Values
+"""
+
+# Product Pages
+PP_REST = "https://pp.engineering.redhat.com/api/latest/"
```

Comment:
The real URL should be set in our internal repository. This public repos should not include any internal stuff, probably not even the products listed below this line.

We can pass configuration in **environment variables** (see the section in `REAMDE.md`).

Reply:
ack